cmd/tke-installer/app/installer/manifests/tke-registry-controller/tke-registry-controller.yaml

```diff
@@ -78,8 +78,17 @@ data:
     registry_config = "/app/conf/registry-config.yaml"
     
     [registry_setting]
-    default_system_chartgroups = ["public"]
+    default_system_chartgroups = ["platform"]
+    chart_paths = ["/app/registry/tencent-charts.tar.gz"]
```

Comment:
what is the difference between place `tencent-charts.tar.gz` to `installer` container and  `tke-registry` container?
what about only add `tencnet chart repo` by default but don't upload chart package?
Currently we only have chart tgz but don't have chart image,  how to handle offline installer,  pack all chart images to tkestack release package is not good approach as well